### PR TITLE
Return the js stacktrace in the DuktapeException message if available.

### PR DESCRIPTION
**Pull request by szurbrigg**

### CODE REVIEW

File: tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java [hunk 1]
```diff
@@ -47,7 +47,31 @@
...
+            + "\n"
+            + "function f2() {\n"
+            + "  nope();\n"       // Line 10.
+            + "}\n");
+      fail();
+    } catch (DuktapeException e) {
+      // The first line is the error type and message.
+      assertThat(e.getMessage()).startsWith("ReferenceError: identifier 'nope' undefined");
+      // Each following line in the stacktrace is <function> <filename>:<linenumber>.
+      assertThat(e.getMessage()).containsMatch("f2 .*:10");
```
Comment (JakeWharton) [inline tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java]: You can chain all these calls

**Comment by JakeWharton** [comment]
Looks great!

File: tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java [hunk 2]
```diff
@@ -47,7 +47,31 @@
...
+            + "\n"
+            + "\n"
+            + "function f2() {\n"
+            + "  nope();\n"       // Line 10.
+            + "}\n");
+      fail();
+    } catch (DuktapeException e) {
+      // The first line is the error type and message.
+      assertThat(e.getMessage()).startsWith("ReferenceError: identifier 'nope' undefined");
+      // Each following line in the stacktrace is <function> <filename>:<linenumber>.
```
Comment (swankjesse) [inline tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java]: Neat! We'll probably want to start allowing users to define their own file names!

**Comment by swankjesse** [comment]
LGTM

File: tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java (hunk 1 above)
Comment (szurbrigg) [inline tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java]: Hmm, not sure I follow.  Can you please give an example?

File: tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java (hunk 2 above)
Comment (szurbrigg) [inline tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java]: Will do.

File: tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java (hunk 1 above)
Comment (swankjesse) [inline tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java]:
```
assertThat(e.getMessage()).containsMatch("f2 .*:10")
    .containsMatch("f1 .*:5")
    .containsMatch("eval .*:2");
```

Comment (szurbrigg) [inline tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java]: I see, thanks.  But `containsMatch()` returns void, unless I have something else messed up?

Comment (JakeWharton) [inline tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java]: Oh! That's unusual. I'll have a look into that upstream.

**Pull request merged**